### PR TITLE
2017 05 test multiselect library

### PR DESCRIPTION
**Pull request by MagdaN**
added multiselect with template tag, but we can remove the template tag if this better doable with crispy forms

### CODE REVIEW

File: apps/ideas/models/abstracts/applicant_section.py [hunk 1]
```diff
@@ -32,8 +32,9 @@ class AbstractApplicantSection(models.Model):
     first_name = models.CharField(max_length=250,
                                   help_text=FIRST_LAST_NAME_HELP)
     last_name = models.CharField(max_length=250)
-    organisation_status = models.CharField(max_length=255,
-                                           choices=ORGANISATION_STATUS_CHOICES
+    organisation_status = MultiSelectField(max_length=255,
```
Comment (slomo) [inline apps/ideas/models/abstracts/applicant_section.py]: This shouldn't be a multi select. You can only have one organisation status.

File: apps/ideas/templatetags/idea_tags.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter(name='is_multi_select_checkbox')
+def is_checkbox(field):
+    return field.field.widget.__class__.__name__ == 'CheckboxSelectMultiple'
```
Comment (slomo) [inline apps/ideas/templatetags/idea_tags.py]: What is called field in the definition is already the widget, or isn't it?

**Review by slomo — CHANGES_REQUESTED** [review]
I think we should definitly fix the issues with organisation type.

**Review by slomo — APPROVED** [review]

**Review by fuzzylogic2000 — APPROVED** [review]
Seems to work fine, but a help text (chose 1 or 2)  for the idea topics would be good.

**Comment by slomo** [comment]
@fuzzylogic2000 Added help texts

**Pull request merged**